tests/conftest.py

```diff
@@ -2,30 +2,100 @@
 import os
 import shutil
 import subprocess
+import sys
+import tempfile
+import time
 import uuid
 from unittest import mock
 
 import pytest
+import requests
 from opentelemetry import trace as trace_api
 
 import mlflow
 from mlflow.tracing.display.display_handler import IPythonTraceDisplayHandler
 from mlflow.tracing.export.inference_table import _TRACE_BUFFER
 from mlflow.tracing.fluent import _set_last_active_trace_id
 from mlflow.tracing.trace_manager import InMemoryTraceManager
-from mlflow.tracking._tracking_service.utils import _use_tracking_uri
-from mlflow.tracking.fluent import (
-    _last_active_run_id,
-    _reset_active_model_context,
-    _reset_last_logged_model_id,
-)
 from mlflow.utils.file_utils import path_to_local_sqlite_uri
 from mlflow.utils.os import is_windows
+from mlflow.version import IS_TRACING_SDK_ONLY
 
 from tests.autologging.fixtures import enable_test_mode
+from tests.helper_functions import get_safe_port
+from tests.tracing.helper import purge_traces
+
+if not IS_TRACING_SDK_ONLY:
+    from mlflow.tracking._tracking_service.utils import _use_tracking_uri
+    from mlflow.tracking.fluent import (
+        _last_active_run_id,
+        _reset_active_model_context,
+        _reset_last_logged_model_id,
+    )
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(autouse=IS_TRACING_SDK_ONLY, scope="session")
+def remote_backend_for_tracing_sdk_test():
+    """
+    A fixture to start a remote backend for testing mlflow-tracing package integration.
+    Since the tracing SDK has to be tested in an environment that has minimal dependencies,
+    we need to start a tracking backend in an isolated uv environment.
+    """
+    port = get_safe_port()
+    # Start a remote backend to test mlflow-tracing package integration.
+    with tempfile.TemporaryDirectory() as temp_dir:
+        with subprocess.Popen(
+            [
+                "uv",
+                "run",
+                "--with",
+                "mlflow",
+                "--python",
+                # Get current python version
+                f"{sys.version_info.major}.{sys.version_info.minor}",
+                "mlflow",
+                "server",
+                "--port",
+                str(port),
+            ],
+            cwd=temp_dir,
+        ) as process:
+            print("Starting mlflow server on port 5000")  # noqa: T201
+            try:
+                for _ in range(30):
```

Comment:
what is the typical start time in your tests? If it's <5s, 30s wait-for-terminate seems reasonable. If it's anywhere between 5-20s, let's just use 60 here.